Documentation/ApiOverview/BackendModules/ModuleConfiguration.rst

```diff
@@ -0,0 +1,287 @@
+means the state is fixed and cannot be changed at runtime.
+
+Example: register two backend modules
+=====================================
+
+You can find the following example in
+`EXT:examples <https://github.com/TYPO3-Documentation/t3docs-examples>`__.
+
+Two backend modules are being registered. The first module is based on
+Extbase while the second uses a plain controller.
+
+.. code-block:: php
+   :caption: EXT:my_extension/Configuration/Backend/Modules.php
+
+   use T3docs\Examples\Controller\ModuleController;
+   use T3docs\Examples\Controller\AdminModuleController;
+
+   /**
+    * Definitions for modules provided by EXT:examples
+    */
+   return [
+       'web_examples' => [
+           'parent' => 'web',
+           'position' => ['after' => 'web_info'],
+           'access' => 'user',
+           'workspaces' => 'live',
+           'path' => '/module/page/example',
+           'labels' => 'LLL:EXT:examples/Resources/Private/Language/Module/locallang_mod.xlf',
+           'extensionName' => 'Examples',
+           'controllerActions' => [
+               ModuleController::class => [
+                   'flash','tree','clipboard','links','fileReference','fileReferenceCreate',
+               ],
+           ],
+       ],
+       'admin_examples' => [
+           'parent' => 'system',
+           'position' => ['top'],
+           'access' => 'user',
+           'workspaces' => 'live',
+           'path' => '/module/system/example',
+           'labels' => 'LLL:EXT:examples/Resources/Private/Language/AdminModule/locallang_mod.xlf',
+           'extensionName' => 'Examples',
+           'controllerActions' => [
+               AdminModuleController::class => [
+                   'index',
+               ],
+           ],
+       ],
+   ];
+
+
+Module configuration options
+============================
+
+.. confval:: parent
+
+   :Scope: Backend module configuration
+   :type: string
+
+   If the module should be a submodule, the parent identifier (for example `web`)
+   has to be set here. You can find a
+   :ref:`list of available toplevel modules. <backend-modules-toplevel-module>`
+
+   Extensions can add additional parent modules by defining
+   :confval:`standalone` modules.
+
+.. confval:: path
+
+   :Scope: Backend module configuration
+   :type: string
+   :Default: `/module/<mainModue>/<subModule>`
+
+   Define the path to the default endpoint. The path can be anything, but
+   will fallback to the known  `/module/<mainModue>/<subModule>` pattern,
+   if not set.
+
+.. confval:: access
+
+   :Scope: Backend module configuration
+   :type: string
+
+   Can be `user` (editor permissions), `admin`, or  `systemMaintainer`.
+
+.. confval:: workspaces
+
+   :Scope: Backend module configuration
+   :type: string
+
+   Can be `*` (= always), `live` or `offline`
+
+
+.. confval:: position
+
+   :Scope: Backend module configuration
+   :type: array
+
+   The module position. Allowed values are `top` and `bottom` as
```

Comment:
Allowed values are `before => <identifier>` and `after => <identifier>`. To define modules on top or at the bottom, `before => *` and `after => *` can be used. Using  the `top` and `bottom` values (without key) is deprecated and will be removed in upcoming versions.